tools/check_format.py

```diff
@@ -160,8 +162,8 @@ def checkSourceLine(line, file_path, reportError):
     # comments, for example this one.
     reportError("Don't use <mutex> or <condition_variable*>, switch to "
                 "Thread::MutexBasicLockable in source/common/common/thread.h")
-  if not whitelistedForProdTime(file_path):
-    if 'ProdSystemTimeSource' in line or 'ProdMonotonicTimeSource' in line:
+  if not whitelistedForRealTime(file_path):
+    if 'RealTimeSource' in line or 'RealTimeSystem' in line:
       reportError("Don't reference real-time sources from production code; use injection")
```

Comment:
I realize this isn't in your PR, but I still think we should lose the dash in real-time since it make it sound like a RTOS reference.

Reply:
done